examples/azure/main.tf

```diff
@@ -15,18 +15,20 @@ module "azure" {
   cluster_name = "phoenix"
   domain       = "calculquebec.cloud"
 
-  # Visit https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux
+  # Visit https://azuremarketplace.microsoft.com/en-us/marketplace/apps/almalinux.almalinux-x86_64
   # to contract the free AlmaLinux plan and be able to use the image.
+  # You can also do this via the Azure CLI with
+  #   az vm image terms accept --publisher almalinux --offer almalinux-x86_64 --plan 9-gen2
```

Comment:
For me, going through the web interface didn't really work...I think you have to actually launch an instance to get to the point where you've (implicitly?) accepted the legal terms

Reply:
This may not be true, I really don't know, since I couldn't use the plan anyway (see https://github.com/ComputeCanada/magic_castle/pull/276#issuecomment-1875635725) I'm not  really sure what a normal interface looks like.